WordPress/UITestsFoundation/XCUIApplication+ScrollDownToElement.swift

```diff
@@ -20,18 +20,20 @@ extension XCUIApplication {
 
     /// Scrolls down to element until it becomes hittable.
     /// After that attempts to scroll it to the screen top.
-    func scrollDownToElement(element: XCUIElement, maxScrolls: Int = 5) {
+    func scrollDownToElement(element: XCUIElement, maxScrolls: Int = 10) {
         for _ in 0..<maxScrolls {
-            if element.exists && element.isHittable {
-                element.scrollToTop()
-                break
+            if !element.isFullyVisibleOnScreen() {
+                scrollDown()
```

Comment:
I changed the logic of this extension a bit. It worked for the situation of two cards, one being approximately at the top, and another one at the bottom. With the 3rd added card, the test scrolled past the middle card. Now, `scrollDown` is called more times (10 instead of 5), scroll happens in shorter intervals (scrolling 30% of screen instead of 80%), and it happens with slow velocity, so that there's not "scrolling momentum" taking place afterwards.